File: NapakalakiRuby/lib/prize.rb
# encoding: utf-8
# 
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

class Prize
  
  public
  
  def initialize(treasures, levels) 
    @treasures = treasures
    @level = levels
  end
  
  def get_treasures
    @treasures
  end
  
  def get_levels
    @level
  end

  # Devuelve un string con el buen rollo del monstruo
  def to_s    
    "Prize -> Treasures: #{@treasures}, Level: #{@level}"    
  end
  
end

File: Exámenes/Examen 3/Napakalaki/src/napakalaki/Prize.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package napakalaki;

/**
 *
 * @author caracuel
 */
public class Prize {
    
    // Variables
    private int treasures, level;
    
    // Constructor
    Prize(int treasures, int level){
        
        setTreasures(treasures);
        setLevel(level);
        
    }
    
    // Getters
    public int getTreasures(){
        return treasures;
    }
    
    public int getLevel(){
        return level;        
    }
    
    // Setters
    private void setTreasures(int treasures){
        this.treasures = treasures;
    }
    
    private void setLevel(int level){
        this.level = level;
    }
    
    // Utils
    @Override
    public String toString(){
        return "Treasures = " + Integer.toString(treasures) +
                ", Level = " + Integer.toString(level);
    }
    
}
